sn_node/src/node.rs

```diff
@@ -95,6 +97,7 @@ impl NodeBuilder {
         root_dir: PathBuf,
         owner: Option<String>,
         #[cfg(feature = "upnp")] upnp: bool,
+        relay_server: bool,
```

Comment:
I think this need not be passed as an arg to `NodBuilder::new()`, since this is a builder, we can add a function inside `NodeBuilder` to set the value. And then call it if necessary?

Reply:
The same should've been done for `owner` arg :( can we create a commit for that as well if possible?